docs/comparison.md

```diff
@@ -81,12 +81,14 @@ hoisting all necessary data requirements to a page-wide query.
 |                               | urql                             | Apollo             | Relay             |
 | ----------------------------- | -------------------------------- | ------------------ | ----------------- |
 | React Bindings                | ✅                               | ✅                 | ✅                |
-| React Hooks Support           | ✅                               | ✅                 | ✅ (experimental) |
+| React Concurrent Hooks Support| 🛑                               | ✅                 | ✅ (experimental) |
```

Comment:
```suggestion
| React Concurrent Hooks Support| ✅                               | ✅                 | ✅ (experimental) |
```

Just as stated in the other note; concurrent mode and suspense is supported in `urql`, so I'm more worried that the Apollo entry here is inaccurate, but I'll figure that one out later :)

Reply:
I will wait for your feedback and then I will update the pr 👍

Reply:
So I update that urql supports react concurrent hooks and apollo doesn't?

Reply:
I've just added the above suggestion, since I can directly add that to the batch 😅